back/src/index.js

```diff
@@ -119,6 +121,59 @@ app.get('/comments', (_, res) => {
   });
 });
 
+app.post('/auth', (req, res) => {
+  console.log(req.body);
```

Comment:
No console.log

Reply:
Resolved in 2131c8dc98da6ec64382e2a1e66af6b64050ae9c